### ISSUE TITLE
Fine Tuning

### ISSUE THREAD

**Issue by scottfernandes** [jacobf18/tabular#6]
Is it possible to fine-tune the model, so that it slightly adjusts the conditional distribution it has learned in pretraining?

**Comment by jacobf18** [comment]
We have not yet built a plug-and-play fine-tuning pipeline, but since the model weights are open-sourced, then can definitely be fine-tuned. You could artificially mask values in a dataset and then fine-tune the model to predict them. However, if the observed data is not representative of the missing values, then it might make the model worse on imputation. The ImputePFN class does allow for setting a local checkpoint, so you can always save a fine-tuned checkpoint and use the same interface.
